data/saas/config/greenhouse_harvest_config.yml

```diff
@@ -0,0 +1,63 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Greenhouse Harvest
+  type: greenhouse_harvest
+  description: A sample schema representing the Greenhouse Harvest API integration for Fides
+  user_guide: https://docs.ethyca.com/user-guides/integrations/saas-integrations/greenhouse_harvest
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      label: The Harvest domain. Default harvest.greenhouse.io
+      default_value: harvest.greenhouse.io
+    - name: api_key
+      label: The Harvest API key processed value. This can be generated by navigating from the Greenhouse main page to Configure >> Dev Center >> API Credential Management and taking note of the Harvest API key value. A colon must be appended and the resulting string encoded base64. Please see the Harvest API documentation regarding authentication for more details https://developers.greenhouse.io/harvest.html#authentication
+      sensitive: True
+    - name: greenhouse_user_id
+      label: Your Greenhouse USER id. This value can be seen by any Greenhouse user from the main page by examining the URL when viewing this user. From the Greenhouse main page, navigate to Configure -> Users -> Select the desired user. The URL when viewing the account will look like this https://app8.greenhouse.io/account/users/4054555008/edit where the numeric value is the Greenhouse user ID.
```

Comment:
The `label` value should be the user-friendly version of domain, api_key, and greenhouse_user_id so Domain, Api Key and Greenhouse user ID. The text you have there is better suited for the `description`